tools/devtools/src/page_scripts/owl_devtools_global_hook.js

```diff
@@ -388,7 +388,8 @@
       let targetToKeysToCallbacks;
 
       // Step 1: extract internal values from owl
-      const obj = owl.reactive({}, () => {});
+      const obj =
+        window?.owl?.reactive({}, () => {}) || window.__OWL_DEVTOOLS__.reactive({}, () => {});
```

Comment:
remove the empty callback, it's implicit and there is a performance optimization if you don't specify it. 

I don't think this solution works well with multiple instances of owl (eg one in the main page and one in an iframe) because they each have their own version of reactive that captures different weakmaps, so the toRaw of one owl doesn't work with another owl.

Reply:
There is one hook per instance of owl so I don't think it's really problematic: the iframe's __OWL_DEVTOOLS__ window variable will be unrelated to the main frame's one so even if the code is different, there is no interaction between both

Reply:
For the empty callbacks, i'm not sure about why it matters since Géry did it but it does indeed matter and won't work without it

Reply:
> For the empty callbacks, i'm not sure about why it matters since Géry did it but it does indeed matter and won't work without it
Then it's a bug and we should look into it.

> There is one hook per instance of owl so I don't think it's really problematic: the iframe's OWL_DEVTOOLS window variable will be unrelated to the main frame's one so even if the code is different, there is no interaction between both

Hm, I guess that's true, you would need to have 2 separate instances of owl in the same window which should not be a frequent use case. You should probably extract reactive and toRaw just once at the top level. If you keep the `window.owl` method, it should probably be as a fallback and mention that this is for backwards compatibility with owl <=2.1.2 that doesn't expose these in the devtools hook yet.

Reply:
the empty callback is necessary to make owl go through the non optimized path so the horrible hack just below works. So this PR does not worsen the current situation.

Now, the case with multiple owl versions is still a problem (not with an iframe though, since the iframe will load a separate copy of the owldevtools global hook). I strongly suspect that it does not work now, and that it would require some other changes in the way we expose owl internal values. I guess it should be solved in a later issue